code/__DEFINES/traits.dm

```diff
@@ -68,6 +68,7 @@
 #define TRAIT_HUSK				"husk"
 #define TRAIT_BADDNA			"baddna"
 #define TRAIT_CLUMSY			"clumsy"
+#define TRAIT_CHUNKYFINGERS		"chunckyfingers" //means that you can't use weapons with normal trigger guards.
```

Comment:
"Chuncky" might wanna correct that

Reply:
Eh. It's more unique that way and prevents possible conflicts :tada: